proteus/MeshAdaptPUMI/cMeshAdaptPUMI.cpp

```diff
@@ -305,6 +315,14 @@ int MeshAdaptPUMIDrvr::getSimmetrixBC()
 #endif
   return 0;
 } 
+static int countTotal(apf::Mesh* m, int dim)
+{
+  int total = apf::countOwned(m, dim);
+  PCU_Add_Ints(&total, 1);
+  return total;
+}
+
+#include "PyEmbeddedFunctions.h"
```

Comment:
This may just me being old-fashoned, but I like having all the headers at the top of the file to make it immediately clear about the dependencies.

Reply:
I think that perspective is still in vogue, header was updated.